app/spec/controllers/cbv/entries_controller_spec.rb

```diff
@@ -0,0 +1,84 @@
+require "rails_helper"
+
+RSpec.describe Cbv::EntriesController do
+  describe "#show" do
+    render_views
+
+    it "renders properly" do
+      get :show
+
+      expect(response).to be_successful
```

Comment:
This comment isn't particular to any single contributor, but as a reviewer I would typically expect more assertions for this test i.e. testing that the response body contains the expected content.

```suggestion
expect(response).to be_successful
expect(response.body).to include("Expected content")
```

Reply:
Good point - I had originally put these tests here to test the minimum functionality possible, assuming we would continue to beef them up with more test cases as we developed more functionality. We should certainly continue to do that.